deploy/helm/kubecf/templates/s3-blobstore.yaml

```diff
@@ -0,0 +1,122 @@
+{{- if eq .Values.features.blobstore.provider "s3" }}
+
+apiVersion: "v1"
+kind: "Secret"
+type: Opaque
+metadata:
+  name: var-blobstore-access-key-id
+  namespace: {{ .Release.Namespace | quote }}
+  labels:
+    app.kubernetes.io/instance: {{ .Release.Name | quote }}
+    app.kubernetes.io/managed-by: {{ .Release.Service | quote }}
+    app.kubernetes.io/name: {{ default .Chart.Name .Values.nameOverride | trunc 63 | trimSuffix "-" | quote }}
```

Comment:
Shouldn't this use the [`kubecf.name`](https://github.com/cloudfoundry-incubator/kubecf/blob/d393c89bcf0504f6b9f2509ef59fb31ced692925/deploy/helm/kubecf/templates/_helpers.tpl#L5) macro?

(I'm surprised we don't have a macro for standard labels.)

Reply:
DONE. nice catch.